pkg/plugins/platform/types.go

```diff
@@ -1,5 +1,7 @@
 package platform
 
+import "log"
```

Comment:
I don't actually insist on `logrus`. If it's come across that way I apologize! I do however encourage people to use it all over if they need any of the advanced functionality not in the standard log package (like `.Info()`, `.Debug()`, etc.). Typically I'll switch to logrus without hesitation if I require any of that.